README.md

```diff
@@ -133,7 +157,7 @@ defmodule AddUsers do
 end
 ```
 
-  > #### Cassandra and Scylla Types {: .info}
+  > ### Cassandra and Scylla Types {: .info}
```

Comment:
Can you revert this change, as it's subjective?

Reply:
done